frontend/packages/console-dynamic-plugin-sdk/src/extensions/pages.ts

```diff
@@ -77,5 +80,6 @@ export const isResourceListPage = (e: Extension): e is ResourceListPage =>
 export const isResourceDetailsPage = (e: Extension): e is ResourceDetailsPage =>
   e.type === 'console.page/resource/details';
 
+/** @deprecated - use `console.tab/horizontalNav` */
```

Comment:
Not sure we need this deprecation notice here, since this function is complementary to the extension type (which is already marked as deprecated).

Reply:
I disagree in it not being needed - I think it has huge value to the developers writing code to see it's deprecated and not to write anymore code with this extension. I think this is a pattern more than a "needed in this case" sorta thing.